graylog2-server/src/main/java/org/graylog2/grok/GrokPatternServiceImpl.java

```diff
@@ -98,4 +98,9 @@ public boolean validate(GrokPattern pattern) {
     public int delete(String patternId) {
         return dbCollection.removeById(new ObjectId(patternId)).getN();
     }
+
+    @Override
+    public void deleteAll() {
+        dbCollection.drop();
```

Comment:
While this *currently* has the same effect, I would replace this with a proper call to [`JacksonDBCollection#remove(com.mongodb.DBObject)`](http://mongojack.org/apidocs/org/mongojack/JacksonDBCollection.html#remove(com.mongodb.DBObject)).